### ISSUE TITLE
"Include Scripture Text" doesn't work for XXA book

### ISSUE THREAD

**Issue by mnjames** [sillsdev/ptx2pdf#696]
I want to print just the introduction of a Bible Module in it's output format (the XXA book, not the original spec file).

If I print a regular book and un-check the "Include Scripture Text" button, it prints the intros without the rest of the text. However when I do the same thing for a XXA-G book, it prints everything.

**Comment by markpenny** [comment]
Good point. I don't think we have ever considered what that option _should_ mean for the XXA-G books.

**Comment by mhosken** [comment]
It's also tricky in that what constitutes scripture text and what is explanatory text around the scripture, is more nuanced in a module. We could just cause all scripture references to return nothing.

**Comment by mnjames** [comment]
In general, I expect books printed via the Bible Module option and via the SIngle Book --XXA option to result in exactly the same output. If they're intentionally different, I'm curious as to why. (There are _lots_ of differences in output, not just this specific issue.)

I guess when I read "Include Scripture Text", what my mind interprets that as is "uncheck in order to print only markers which begin with \i". That's the way the Single Book--XXA option behaves.

I understand that "Include Scripture Text" in the context of Bible Modules is a little ambiguous, in part because it could be interpreted as "print everything in the Bible Module which isn't imported with a \ref marker". But I'd argue against that being the correct interpretation; in particular, if the code was `\ref MAT 1:0` I'd expect it to print even if "Include Scripture Text" was unchecked.

So my personal option would be that the Bible Module behavior should be changed to mimic the Single Book--XXA option, and maybe another line added to the "Include Scripture Text" tooltip which says something like "In the context of Bible Modules, turning off Include Scripture Text prevents any markers which don't begin with 'i' from printing, regardless of whether they are scripture or not."

**Comment by markpenny** [comment]
@mnjames Where are we up to on this issue? Is it one of those that got fixed during the workshop, but never closed? Or is it still an issue that we need to work on?

**Comment by mnjames** [comment]
In my mind it's not fixed, but if the team decides it's not going to be worked on I won't be upset.

I can work around what I was trying to do with a fairly simple regex rule in changes.txt, so I don't need to use the UI to do it.

Reading back through my write-up, it sounds like if "Include Scripture Text" was un-checked, that if I printed XXA then only the intros printed and not the Scripture. But if I did the same thing using the Bible Module directly, that it printed everything.

Oddly enough, testing today both methods result in the same behavior--everything is printed. On the one hand, that means that printing via XXA and via Bible Module has the same result, which I strongly believe should be the case. On the other hand, it means that now _neither_ method now results in the output that I expected.
